### PR TITLE
RE2022-163: Data processing operations goals first cut

### PR DESCRIPTION
**Pull request by MrCreosote**

### CODE REVIEW

File: design/data_processing_operations.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+# Data operations processing design
+
+## Document purpose
+
+Lay out goals / designs for KBase Collections data processing in the short, middle and long term.
+Said goals / designs will necessarly get more vague the further out in time we go.
```
Comment (Tianhao-Gu) [inline design/data_processing_operations.md:6]: *necessarily

File: design/data_processing_operations.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+## Short term goals
```
Comment (Tianhao-Gu) [inline design/data_processing_operations.md:8]: what timeframes are considered short term vs middle term?

**Review by Tianhao-Gu — APPROVED** [review]
short/middle tasks are all reasonable. Wondering if we should have a cut off line for some event like renewal review, RSV or maybe Thanks giving/Labor day.

File: design/data_processing_operations.md (hunk 2 above)
Comment (MrCreosote) [reply design/data_processing_operations.md:8]: I was thinking

Short term = should be working on it now
Medium term = Most likely prior to RSV
Long term = post RSV, maybe by quite a while

Comment (Tianhao-Gu) [reply design/data_processing_operations.md:8]: 👍

File: design/data_processing_operations.md (hunk 1 above)
Comment (MrCreosote) [reply design/data_processing_operations.md]: Fixed

**Review by Tianhao-Gu — APPROVED** [review]
LGTM

**Pull request merged**